prime-router/metadata/schemas/CA/ca-scc-covid-19.schema

```diff
@@ -0,0 +1,205 @@
+name: "ca-scc-covid-19"
+topic: "covid-19"
+description: "CSV download schema for Santa Clara County Public Health Department"
+trackingElement: filler_order_id
+basedOn: "covid-19"
```

Comment:
If we have a link for their schema it's good to put the `referenceUrl` at the top.